stories/CurrentRefinements.stories.js

```diff
@@ -60,24 +60,69 @@ storiesOf('ais-current-refinements', module)
       transformItems(items) {
         return items.map(item =>
           Object.assign({}, item, {
-            label: item.label.toUpperCase(),
+            attribute: item.attribute.toLocaleUpperCase(),
```

Comment:
You can use `label` rather than `attribute`.

Reply:
```suggestion
            attribute: item.label.toLocaleUpperCase(),
```